### DISCUSSION TITLE
How to properly cache the same image file under different names

### DISCUSSION THREAD

**Discussion by willmero** [vercel/community#110] [Help] [answered]
Hello community! 

I have a situation where I have a base placeholder image that will have multiple named instances of the same image. Example:
Root image: (placeholder.png)

placeholder1.png = (placeholder.png)
placeholder2.png  = (placeholder.png)
placeholder3.png = (placeholder.png)

Each placeholder[x].png image is the exact same file as the base PNG image.

What I'm wondering about, is if the Vercel image caching mechanism would be smart enough to realize that it could serve up the Root image automatically from the edge servers if it saw that it was the same file as placeholder[x].png?

**Comment by Pieparker** [comment, marked as answer]
Hey there! In short: different file name, different checksum, different cache keys. It'd be better to consider why you need `placeholder[x].png` and whether there's an alternate method of using the same placeholder which doesn't rely on a file name change.
